clambcc/clambc-compiler.py

```diff
@@ -764,6 +764,24 @@ def _process_long_opt(self, rargs, values):
     def getPassthrough(self):
         return self.passthrough
 
+def printVersion():
+    compilerVersion = "Not written"
+    fileName = os.path.join("headers", "clambc-version.h")
+    if os.path.isfile(fileName):
+        f = open(fileName)
+        lines = f.readlines()
+        f.close()
+
+        for i in range(0, len(lines)):
+            line = lines[i].rstrip()
+            m = re.search('CLAMBC_VERSION "(.*)"', line)
+            if m:
+                compilerVersion = m.group(1)
+                break
+
+    #TODO: Print information for target version (after clamav installs the headers) 
```

Comment:
Printing the latest supported clamav version based on the clamav-generated headers is a related but separate issue. Once clambcc has it's own version number unique from clamav, we may want to have this function print both the clambcc-version and also the latest supported clamav-version.   We could even do that now, although the version #'s will be the same.